velox/docs/functions/spark/string.rst

```diff
@@ -150,6 +150,12 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: translate(string, matchStr, replaceStr) -> varchar
+    Returns a new translated string. It translates the character in ``string`` by a
```

Comment:
Would you share a link to Spark's documentation for this function?

Reply:
A simple doc [here](https://spark.apache.org/docs/latest/api/sql/index.html#translate). Source code comment [here](
https://github.com/apache/spark/blob/1a5b9f295a623901100b6f41e785138fba0bc156/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala#L863).
Assume no need to paste the links in velox doc.